hudi-metrics-common/src/main/java/org/apache/hudi/metrics/config/HoodieMetricsCloudWatchConfig.java

```diff
@@ -38,20 +39,37 @@
   public static final ConfigProperty<Integer> REPORT_PERIOD_SECONDS = ConfigProperty
           .key(CLOUDWATCH_PREFIX + ".report.period.seconds")
           .defaultValue(60)
-          .sinceVersion("0.10.0")
```

Comment:
can we revert any unintentional changes. tough to review just the required changes alone.